binaries/scheduler/main.go

```diff
@@ -21,9 +23,9 @@ func main() {
 	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
 	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
 	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	parser.Report[""].(*config.DefaultReportConfig).HttpAddr = fmt.Sprintf("localhost:%d", *httpPort)
```

Comment:
This looks pretty good, but does mean that if you want to set a report config, you can't... oy. Lacking real DI is a shame. Whatever you think is best.